nuget/helpers/lib/NuGetUpdater/NuGetUpdater.Core.Test/Update/UpdateWorkerTests.Sdk.cs

```diff
@@ -249,7 +249,7 @@ await TestUpdateForProject("Some.Package", "9.0.1", "13.0.1",
                     MockNuGetPackage.CreateSimplePackage("Some.Package", "9.0.1", "net8.0"),
                     MockNuGetPackage.CreateSimplePackage("Some.Package", "13.0.1", "net8.0"),
                     // necessary for the `net8.0-windows10.0.19041.0` TFM
-                    new("Microsoft.Windows.SDK.NET.Ref", "10.0.19041.34", Files:
+                    new("Microsoft.Windows.SDK.NET.Ref", "10.0.19041.45", Files:
```

Comment:
This version was changed since RC1 by https://github.com/dotnet/sdk/pull/41936, and it's going to change again with https://github.com/dotnet/sdk/pull/43889 to `.46`, so will probably need changing next month when updating to `9.0.100`.